Makefile

```diff
@@ -139,7 +139,7 @@ E2E_CONF_FILE  ?= ${E2E_DIR}/config/nutanix.yaml
 ARTIFACTS ?= ${REPO_ROOT}/_artifacts
 SKIP_RESOURCE_CLEANUP ?= false
 USE_EXISTING_CLUSTER ?= false
-GINKGO_NOCOLOR ?= false
+GINKGO_NOCOLOR ?= true
```

Comment:
why did you remove color by default ? i like color

Reply:
when run thru prow job, color was adding lots of characters which was causing parsing issues in prow. you can enable it by passing this value when u run as follows
<pre>
make test-e2e GINKGO_NOCOLOR=true
</pre>

Reply:
maybe it is more easy to set once the GINKGO_NOCOLOR to true in your prow workflow versus asking every user who launch it manually to set it to false